crm_phonecall/models/crm_phonecall.py

```diff
@@ -69,6 +69,8 @@ class CrmPhonecall(models.Model):
     date_closed = fields.Datetime(string="Closed", readonly=True)
     date = fields.Datetime(default=lambda self: fields.Datetime.now())
     opportunity_id = fields.Many2one(comodel_name="crm.lead", string="Lead/Opportunity")
+    direction = fields.Selection([("in", "In"), ("out", "Out")])
```

Comment:
This should be defaulted to `out`, which is probably the default use case in crm_phonecall.
I'd also make it a required field